doc/dev/deimos_radec_object.rst

```diff
@@ -0,0 +1,119 @@
+.. include:: ../include/links.rst
+
+.. _deimos_radec_object_report:
+
+RA, Dec and object name assignment to DEIMOS extracted spectra
+==============================================================
+
+Version History
+---------------
+
+=========   ================   =========== ===========
+*Version*   *Author*           *Date*      ``PypeIt``
+=========   ================   =========== ===========
+1.0         Debora Pelliccia   25 Jan 2021  1.3.1dev
+=========   ================   =========== ===========
+
+----
+
+Basics
+------
+
+The procedure used to assign RA, Dec and object name to each DEIMOS 1D extracted spectrum
+is performed during the object extraction procedure described in :ref:`extraction`.
+
+
+Procedure
+---------
+
+RA, Dec and object name assignment is primarily performed by
+:func:`pypeit.slittrace.SlitTraceSet.assign_maskinfo`. This function, first, reads in the slitmask
+design information stored in the :class:`~pypeit.slittrace.SlitTraceSet` datamodel (see
+:ref:`deimos_slitmask_ids_report` for a description on how the slitmask design matching is performed,
+and :ref:`master_slits` for a description of the provided information and for a way to visualize them).
+
+Then, :func:`~pypeit.slittrace.SlitTraceSet.assign_maskinfo` measures the distance in pixels (converted
+then in arcsec) between the traced object and the center of the slit and, using the coordinates of the
+slit center available from the slitmask design, estimates the coordinates of the extracted spectrum.
+
+``PypeIt`` goes through all the slits in a selected detector and for each extracted spectrum checks if
+the measured coordinates are within a certain tolerance (see `Application`_ for details on how to control the
+value of this parameter) of the expected coordinates for the targeted object. If the answer is yes, the 1D 
+spectrum `RA`, `DEC` and `MASKDEF_OBJNAME` are updated with the coordinates and name of the
```

Comment:
Same thing here about the single vs. double quote characters.

Reply:
Thanks.